modules/tunnels.sh

```diff
@@ -0,0 +1,72 @@
+function tunnels {
+  checkroot
+  case "$1" in
+    pagekite-install)
```

Comment:
Need to add checks to make sure that `email` and `sitename` are populated.

Reply:
doesn't checkargn already take care of this?

Reply:
Checkargn will count the number of arguments and see if the number of arguments provided surpasses that limit or not.

Reply:
done